docs/classifier_example/classification_v3_4_data_points.ipynb

```diff
@@ -0,0 +1,793 @@
+      "text/plain": [
+       "<Figure size 576x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "## 2 points distance-based classifier ##\n",
+    "\n",
+    "# Set-up a new connection with qi_backend:\n",
+    "def initialize_qi_backend(project_name = \"distance_based_classifier\"):\n",
+    "    compiler_engines = restrictedgateset.get_engine_list(one_qubit_gates=\"any\", two_qubit_gates=(CNOT, CZ, Toffoli))\n",
+    "    compiler_engines.extend([ResourceCounter(), ManualMapper(lambda x: x)])\n",
+    "    qi = QuantumInspireAPI(\n",
+    "        r'https://api.quantum-inspire.com/',\n",
+    "        authentication,\n",
+    "        project_name = project_name  # Set project name to save projects\n",
+    "    ) \n",
+    "    qi_backend = QIBackend(num_runs = 1, quantum_inspire_api=qi) # set num_runs = 1 for true probability distribution\n",
```

Comment:
maybe it's just my machine, but this line does not fit in a cell

Reply:
Depends on the width, I'll make some adjustments to keep the code the normal python length.